### PR TITLE
Create markup for appmenu

### PR DESCRIPTION
**Pull request by Pedrael**

### CODE REVIEW

File: src/components/Timedisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  const [time, setTime] = useState<Date>(new Date())
+  useEffect(() => {
+    const timerID = setInterval(() => tick(), 1000)
+
+    return () => {
+      clearInterval(timerID)
+    }
+  }, [])
+
+  const tick = () => {
```
Comment (ReidenXerx) [inline src/components/Timedisplay.tsx]: I think is not necessary to move it in separate function
Use inline

File: src/i18n.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ i18n.use(initReactI18next).init({
     },
   },
   lng: 'ru',
-  fallbackLng: 'en',
+  fallbackLng: ['en', 'ru'],
```
Comment (ReidenXerx) [inline src/i18n.ts:18]: Are u sure that need to add all languages to fallback?

**Review by ReidenXerx — CHANGES_REQUESTED** [review]

File: src/i18n.ts (hunk 2 above)
Comment (Pedrael) [reply src/i18n.ts:18]: I`ll change it to a more appropriate variant in future pr if I find it. For now it works

File: src/components/Timedisplay.tsx (hunk 1 above)
Comment (Pedrael) [reply src/components/Timedisplay.tsx]: fixed

**Review by ReidenXerx — APPROVED** [review]

**Pull request merged**